sm_modified_cnn/create_dataset.sh

```diff
@@ -0,0 +1,16 @@
+mkdir data
```

Comment:
Please make this script executable, i.e. `chmod +x create_dataset.sh`.

Please add a shebang to the script.